test/tests/functional/pbs_cpuset.py

```diff
@@ -87,6 +87,11 @@ def test_reliable_job_startup_on_cpuset(self):
         The released numa node can be used in another job.
         """
 
+        # Check that there are at least a natural vnode + 2 numa vnodes
+        nodeinfo = self.server.status(NODE)
```

Comment:
Shouldn't this be VNODE? Or is there no difference

Reply:
Either NODE or VNODE can be used here.